integration-cli/docker_cli_daemon_test.go

```diff
@@ -32,7 +32,7 @@ import (
 	testdaemon "github.com/docker/docker/internal/test/daemon"
 	"github.com/docker/docker/opts"
 	"github.com/docker/docker/pkg/mount"
-	"github.com/docker/go-units"
+	units "github.com/docker/go-units"
```

Comment:
argh. I need to figure out how to prevent my IDE from adding these unneeded aliases 😂